portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewFields.testcase

```diff
@@ -1481,6 +1481,320 @@ definition {
 		}
 	}
 
+	@description = "LPS-187522. Allow administrators to configure client extension filters on dataset views."
+	@priority = 5
+	test ClientExtensionFrontendDataSetFilterCreationCancellationTest {
```

Comment:
This test name is not correct. You don't need to add "test" to a test name, we know that it is a test :P This test name should be something like "ClientExtensionFilterCancelButton"